### ISSUE TITLE
Can we have a "public" switch for snapshots?

### ISSUE THREAD

**Issue by robertgarrigos** [backdrop-contrib/demo#30]
There is some interest in having a switch of snapshots for an authenticated user, this is, a normal authenticated user could be able to switch ONLY between snapshots, nor creating nor editing them. Let's work it out? Any thoughts/ideas on this? Could this be just a block with a select item and a button?

**Comment by izmeez** [comment]
I'm not sure of a good use case for this. It might work if the site was essentially a static site with several static states. If it is a dynamic site with changing content then snapshots really work best in a development or testing environment. The site could get messed up if authenticated users could switch between snapshots while the site is dynamic.

**Comment by robertgarrigos** [comment]
I agree that it would not be of use for a dynamic site, but for a demonstration site. For example, templates demonstration where anonymous users could switch templates.
